tests/PHPStan/Analyser/nsrt/count-type.php

```diff
@@ -44,12 +44,12 @@ public function doFooBar(
 		if (count($arr) == $maybeZero) {
 			assertType('array', $arr);
 		} else {
-			assertType('non-empty-array', $arr);
+			assertType('array', $arr);
 		}
 		if (count($arr) === $maybeZero) {
 			assertType('array', $arr);
 		} else {
-			assertType('non-empty-array', $arr);
```

Comment:
I think I did a mistake here in previous PR, as the array can still be empty in the else branch

https://3v4l.org/409Jp